lib/sidekiq/form526_backup_submission_process/processor.rb

```diff
@@ -360,6 +360,7 @@ def get_uploads
         uploads = submission.form[FORM_526_UPLOADS]
         uploads.each do |upload|
           guid = upload['confirmationCode']
+          # [wipn8923] update to use flipper (probably need new SupportingEvidenceAttachment for LH specific inheritance
```

Comment:
Yeah I think we will need to do this anywhere  SupportingEvidenceAttachment's are referenced in a form 526 context